pkgs/applications/networking/remote/x2goclient/default.nix

```diff
@@ -0,0 +1,31 @@
+{ fetchurl, stdenv, nxproxy, qt4, openldap, libssh, libXpm, cups, makeWrapper }:
+
+stdenv.mkDerivation rec {
+  version = "4.0.3.1";
+  name = "x2goclient-${version}";
+
+  src = fetchurl {
+    url = "http://code.x2go.org/releases/source/x2goclient/x2goclient-4.0.3.1.tar.gz";
+    md5 = "ef9a20ef96f7c31cc20ece9ebbf1e007";
```

Comment:
Usual convention is to include `${name}` in the `url` when possible, so it doesn't also need updating on every version bump. Also, I think sha256 hashes are preferred (easily copied from `nix-prefetch-url`).